src/index.js

```diff
@@ -0,0 +1,17 @@
+const scores = [
+  { name: 'Jim', score: '30' },
+  { name: 'Ntare', score: '40' },
+];
```

Comment:
- [ ] [OPTIONAL] good job on the arrow function 👍 however in order for your styling to work you need to import the CSS file at the top of your Javascript file as well as in the example below:
```
import './style.css'
```